src/main/java/chatbot/task/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package chatbot.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class that inherits from Task.
+ * 
+ * @var deadline Representing deadline.
+ * 
+ * @author Owen Yeo
+ */
+public class Deadline extends Task{
+
+    private LocalDateTime deadline;
+
+    private String originalString;
+
+    /**
+     * Constructor for a deadline object.
+     * 
+     * @param label Descriptor for the task with deadline
+     * @param deadline String representing deadline
+     */
+    public Deadline(String label, String deadline) {
+        super(label);
+        this.deadline = LocalDateTime.parse(deadline, DateTimeFormatter
+            .ofPattern("yyyy-MM-dd HHmm"));
+        this.originalString = deadline;
+    }
+
+    /**
+     * 
+     */
+    @Override
+    public String toSaveString() {
+        return "D " + super.toSaveString() + " | " + originalString;
+    }
+
+    /**
+     * 
+     */
```

Comment:
Perhaps a JavaDoc comment here? Since the methods are overriden, I think removing the comment is fine too!